### ISSUE TITLE
When I visit your site there are no instructions for the user

### ISSUE THREAD

**Issue by edgarfinn** [FACN1/week7-rm-rf-mario#31]
Its tricky to understand how to use your site on first arrival at your site. 

On a UX tip, I would provide some basic instructions for first-time visitors :)

**Comment by edgarfinn** [comment]
Same also for the update page. Just to add clarity

**Comment by m4v15** [comment]
I agree, it's quite unclear what clicking the buttons is going to do - when I click the shekhel button I'd probably expect to see the table? add would surely be a + sign or something
